server/src/main/java/org/elasticsearch/persistent/PersistentTasksCustomMetaData.java

```diff
@@ -403,8 +397,9 @@ public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params xPa
                         if (params != null) {
                             builder.field("params", params, xParams);
                         }
-                        if (status != null) {
-                            builder.field("status", status, xParams);
+                        if (state != null) {
```

Comment:
we never use xcontent for cross node communication. I think we can use state and be tolerant of reading status when parsing? (until 7.0)

Reply:
Yes, I think it should work. Rollup/ML jobs can be snapshotted as part of the global cluster state, but if it still parses `status` as a deprecated field it should be OK.